NOTICE

```diff
@@ -0,0 +1,10 @@
+The Ibex Project
+Copyright 2024 lowRISC contributors.
+
+This product includes hardware and/or software developed as part of the
+Ibex(R) (https://github.com/lowRISC/ibex) and OpenTitan(R) projects.
```

Comment:
Do we want (R) or ®?

Reply:
Unfortunately ® is rejected by our CI system at the moment as it is not ASCII.